src/test/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReaderTest.kt

```diff
@@ -0,0 +1,69 @@
+package org.wycliffeassociates.otter.common.domain.resourcecontainer.project.markdown
+
+import junit.framework.TestCase
+import org.junit.Test
+import org.wycliffeassociates.otter.common.collections.tree.OtterTree
+import org.wycliffeassociates.otter.common.collections.tree.OtterTreeNode
+import org.wycliffeassociates.otter.common.domain.resourcecontainer.project.IProjectReader
+import java.io.File
+import java.nio.file.Paths
```

Comment:
I was actually surprised and since I last tried to use java.nio, some of it has actually been added to Android. However, it was introduced in 8.0; but we typically have to target 4.4 or 5.0.

Would you mind if we moved this test to the JVM repo?

Reply:
I can move it, but would it be ok if I just remove this NIO import and change line 12 to this instead: 
```private val pwd = File(System.getProperty("user.dir"))```
You still won't want to *run* this test on Android, since under Android `user.dir=/` which would make this test take forever.

Reply:
I'm sure you wouldn't even get the file permissions to even be able to do it.

I suppose if we did get to porting to Android and did want to run tests, we'd probably be more concerned with more Android centric things; but I believe there are services to throw tests at a bunch of Android devices and get the results back lol.

Either is fine with me though.

Reply:
Removed NIO import.